### PR TITLE
Icon updater

### PR DESCRIPTION
**Pull request by jacobmitchell04**
* Added mass icon updater.
* Fixed manual icon updater.

### CODE REVIEW

File: src/Dynamic/lists/iconupdater.pug [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    section.section
+        .container
+            .columns
+                .column
+                    .card
+                        .card-content
+                            h1.title.is-size-4 #{__('site_name')} Icon Updater
+                    .card
+                        .card-content
+                            #{lists.map((m) => m).join('<br>')}
```
Comment (MattIPv4) [inline src/Dynamic/lists/iconupdater.pug]:
```suggestion
                            #{lists.map((m) => m).join('<br/>')}

```
Self-closing tag, ending line break.

File: src/Routes/Lists.js [hunk 2]
```diff
@@ -70,6 +71,25 @@ class ListsRoute extends BaseRoute {
...
+                const lists = await this.db.select().from('lists').where({ display: true, defunct: false });
+                    let messages = [];
+                    for (const list of lists) {
+                        try {
+                            const update = await updateIcon(this.client, this.db, list);
+                            messages.push(list.id + ' - ' + update);
+                        } catch (e) {
+                            messages.push(list.id + ' - ' + e);
+                        }
+                    }
```
Comment (MattIPv4) [inline src/Routes/Lists.js]: Why is this indented?

File: src/Routes/Lists.js [hunk 3]
```diff
@@ -415,7 +435,6 @@ class ListsRoute extends BaseRoute {
                 res.status(500).render('error', { title: 'Database Error' });
             }
         });
-
```
Comment (MattIPv4) [inline src/Routes/Lists.js:418]: Unrelated change?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: src/Routes/Lists.js (hunk 2 above)
Comment (jacobmitchell04) [reply src/Routes/Lists.js]: Fixed.

**Review by MattIPv4 — COMMENTED** [review]
This doesn't appear to be working as expected in dev, not sure why.
image

**Comment by MattIPv4** [comment]
Closing as discussed in Discord.

**Pull request closed without merging**